### PR TITLE
Fix purge command

### PR DESCRIPTION
**Pull request by aintgoin2goa**
remove proxy url and add a test

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,5 +11,9 @@ else
 endif
 	./bin/next-build-tools.js verify --skip-layout-checks
 
+test:
+	mocha ./test/
```
Comment (matthew-andrews) [inline Makefile]: not necessary to specify i think test is deafult

**Comment by matthew-andrews** [comment]
i do believe this pr closes #71.  Probably means we need a new ticket “More tests!”, though

**Comment by aintgoin2goa** [comment]
I think you might be pushing it to stay this completes closes that!  I could look at adding some more tests if you like (when I have the time).  Can I merge this?

**Comment by matthew-andrews** [comment]
yep merge away.  don't worry about adding tests unless you need to fix a bit

**Pull request merged**